### DISCUSSION TITLE
Selection Coords and size should stay even after mouse release

### DISCUSSION THREAD

**Discussion by dphaldes** [Orama-Interactive/Pixelorama#739] [Ideas]
Selection start and end coordinates are show when selection(Mouse down) but they disappear after mouse has been release.
![image](https://user-images.githubusercontent.com/51489635/185691157-715d2d2b-f6f9-4b0f-8bd7-fa5e9935c40f.png)

I would like if the details are shown as long as the selection exists.

**Comment by Variable-ind** [comment]
Cause then they are shown in tool properties
And the end point is (start + size)

**Reply by dphaldes** [reply]
Yes but then I have to look at 2 different places : One while selecting and one after selecting.

**Comment by OverloadedOrama** [comment]
Not sure if I agree. The size is already being shown in the tool settings as Variable mentioned, and it can be edited from there too. I don't think there's any need to show the same information twice. Also, what if there is an already active selection and the user creates another one, or draws a shape with the rectangle/ellipse tools? I feel like it would confuse users even further.

**Reply by dphaldes** [reply]
> I don't think there's any need to show the same information twice.

This might be true but it is lacking consistency. Either show the information throughout selection process on the top of the window or in the tool options panel . While working on a project which needed me to measure exact pixels, I found myself constantly looking at different places to get my coordinates.

> Also, what if there is an already active selection and the user creates another one, or draws a shape with the rectangle/ellipse tools? I feel like it would confuse users even further.

This can easily be solved with better positioning and labeling the contents. To be frank,  the current implementation also lacks description. I would have never been able to guess that those were selection coordinates and size in the brackets
